File: app-backend/src/main/java/rs/ac/singidunum/appbackend/models/UserModel.java
package rs.ac.singidunum.appbackend.models;

import lombok.Data;

import java.time.LocalDate;
import java.util.List;

@Data
public class UserModel {
    private String id;
    private String firstName;
    private String lastName;
    private String email;
    private String phone;
    private AddressModel address;
    private List<String> favourites; //NOTE: niz kategorija koje su omiljene korisniku
    private String username;
    private String password;
    private LocalDate dateCreated;
}

File: BIS-project/src/app/models/userModel.ts
import { AddressModel } from "./addressModel";

export class UserModel {
    id?: string = '';
    firstName?: string = '';
    lastName?: string = '';
    email?: string = '';
    phone?: string = '';
    address? = new AddressModel();
    favourites?: string[] = []; //NOTE: niz kategorija koje su omiljene korisniku
    username: string = '';
    password: string = '';
    dateCreated?: Date;
}
